makeservices/makemysql

```diff
@@ -6,6 +6,18 @@ fi
 
 PASS=$(sudo -u mysql /opt/share/utils/makeservices/makemysql-real | tee /dev/tty | tail -n 1 | grep -Po '(?<=: )([0-9a-zA-Z]){24,}$')
 
-echo 'Changing WordPress database password'
-cd ~/public_html/
-wp config set DB_PASSWORD "$PASS" > /dev/null
+# Check if wp is installed with wp-cli
+# And change password if installed with easywp
+# But change password only if installed with easywp is a wrong statement
+# Use --ignore-wp flag to skip this process
+if (wp core is-installed --quiet --path="$HOME/public_html" > /dev/null 2>&1) &&
+        ([ $# -ne 1 ] || ( [ $# -eq 1 ] && [ ! "$1" = '--ignore-wp' ])); then
+    echo "WordPress installation detected, changing WordPress mysql password for you."
+    wp config set DB_PASSWORD "$PASS" > /dev/null 2>&1
+fi
+# Added for backward compatibility - the last line will be
+# extracted by "tail -n 1" and the mysterious "grep"
+# will extract the password after the colon
+# So that it simulate the same behaviour as the original makemysql
+# which output the same thing as in makemysql-real
```

Comment:
```
# So that it simulate the same behaviour as the original makemysql
# which output the same thing as in makemysql-real
```
I would remove these comments; the comments in the previous lines about easywp already tell me everything I need to know.